introduction/environment.md

```diff
@@ -0,0 +1,97 @@
+# Environment setup
+
+The goal of this guide is to setup whole development environment,
+so you can run your first hello world application.
+
+## node.js
+
+Although you might decide to never touch backend, you still need 
+[node.js](https://nodejs.org/en/) to it's job - compile your source codes into single file, 
+automatically reload your app when something has changed, run tests and much more.
+
+## Setup
+
+Install node.js if you haven't done so:
+
+**macOS:**
+
+```bash
+brew install node@6
```

Comment:
I don't have high trust in marriage between latest node & win platform, but I will think about it.